src/js/main.js

```diff
@@ -69,44 +69,40 @@
         })
     });
 
-    module.controller("roomModelViewController", ["roomModel", "constant", "$scope", function(roomModel, constant, $scope) {
-
-        var scene = roomModel.scene;
-
-        /////// Show labels
-        $scope.showLabels = true
+    module.factory("roomModelViewService", ["roomModel", "constant", function(roomModel, constant) {
+        // Support
+        var link = document.createElement('a');
+        link.style.display = 'none';
+        document.body.appendChild(link); // Firefox workaround, see #6594
 
```

Comment:
changed this controller to a factory so we can call the function from the dialog controller